README.md

```diff
@@ -20,7 +20,7 @@ Usage
 -----
 
 * SendGrid has done a [great
-  tutorial](http://blog.sendgrid.com/receiving-email-in-your-rails-app-with-griddler/)
+  tutorial](http://blog.sendgrid.com/blog/receiving-email-in-your-rails-app-with-griddler/)
```

Comment:
Looks like it redirects to the naked domain:

https://sendgrid.com/blog/receiving-email-in-your-rails-app-with-griddler/